src/AutoWrapper/AutoWrapperExcludePaths.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AutoWrapper
+{
+    public enum ExcludeMode
+    {
+        Strict = 1,
+        StartWith = 2,
+        Regex = 3
+    }
+
+    public class AutoWrapperExcludePaths
```

Comment:
Can we make this class in singular form?

Reply:
sure, I will fix it.